cmd/podman/system/connection/add.go

```diff
@@ -76,6 +78,10 @@ func init() {
 	flags.StringVar(&cOpts.UDSPath, socketPathFlagName, "", "path to podman socket on remote host. (default '/run/podman/podman.sock' or '/run/user/{uid}/podman/podman.sock)")
 	_ = addCmd.RegisterFlagCompletionFunc(socketPathFlagName, completion.AutocompleteDefault)
 
+	farmFlagName := "farm"
```

Comment:
I vote to not add this flag yet.  Build farm isn't usable and there's a risk this commit makes it into 4.7 without full buildfarm support.

Reply:
We could make it a hidden option for now, and then allow developers to iterrate and play with it, while it is being developed. we can also fire up CI/CD tests to make sure that no one breaks it.

Reply:
I have marked it as hidden and have tests for it as well. I have added it to the docs, should I remove it from there as it is hidden or it is fine to leave it in the docs?

Reply:
Hiding is an option as well :+1:

Reply:
Flag should be --buildfarm

Reply:
I think we should stick with --farm as that is what we are calling it in the table and also to keep it general if we ever think of a use case for farms other than builds in the future.

Reply:
I am OK with `--buildfarm` as it's symmetric to the `podman buildfarm` commands.  Maybe it's worth renaming `s/farm/buildfarm` in containers.conf.